site-src/v1alpha2/guides/tcp.md

```diff
@@ -14,17 +9,50 @@ distribute the traffic with the following rules:
 - All TCP streams on port 8090 of the Gateway are forwarded to port 6000 of
   `my-bar-service` Kubernetes Service.
 
-Please note the following:
+In this example two `TCP` listeners will be applied to the [Gateway][gateway]
+in order to route them to two separate backend `TCPRoutes`, note that the
```

Comment:
```suggestion
in order to route them to two separate `TCPRoute` resources. Note how the
```
shorter sentences
removed backend because that is another logical term in the API and has potential for confusion.